.gitignore

```diff
@@ -1,6 +1,7 @@
 *.egg-info
 *.py[co]
 *.sw[op]
+.secrets.baseline
```

Comment:
`.secrets.baseline` felt like it should only be in a local copy. I might be mistaken.

Reply:
Ahh, that is confusing, sorry about that, it is purposefully meant to be included in the Git repo though.

Reply:
Thanks, I re-added it :)